ui/00_message/common/spot_name.xml

```diff
@@ -2117,6 +2156,7 @@
   <message>
     <key>SPOT_NAME_949</key>
     <original>水底の古道</original>
+    <translation>Watersdeep Old Road</translation>
```

Comment:
Since this is an old road, it's probably a dry riverbed. Would be best to check against the description (if available) and the location in-game.